syntax = "proto3";

package fraud_detection;

service FraudDetectionService {
    rpc CheckFraud (FraudDetectionRequest) returns (FraudDetectionResponse) {}
}

message FraudDetectionRequest {
    string country = 1;
    string city = 2;
}

message FraudDetectionResponse {
    bool is_fraudulent = 1;
    string reason = 2;
}